packages/react-experiments/src/components/SelectedItemsList/SelectedPeopleList/Items/SelectedPersona.tsx

```diff
@@ -145,7 +147,9 @@ const SelectedPersonaInner = React.memo(
       [selected, isValid, theme],
     );
 
-    const coinProps = {};
+    const defaultPersonaSize = PersonaSize.size32;
+    const personaSize = size !== undefined ? size : defaultPersonaSize;
+    const coinProps = { size: personaSize };
```

Comment:
same comment here about this already being somewhere, I don't think we want to reset it. actually we can probably remove it from being manually set here at all.

Reply:
This no longer applies, since this code is no longer necessary (and was deleted). Resolving.